lib/components/create-program-modal/index.tsx

```diff
@@ -0,0 +1,27 @@
+import ProgramForm from '../program-form';
+import PopUp from '../pop-up';
+
+type CreateProgramModalProps = {
+  modalManagement: {
```

Comment:
Not sure what ur asking. `isOpen`, `onOpen`, and `onClose` are just the fields of `modalManagement`. That's how ChakraUI's modals work.

Reply:
I meant why not:
type CreateProgramModalProps = {
    isOpen: boolean;
    onOpen: () => void;
    onClose: () => void;
}

Reply:
Cuz I feel like that's not very scalable. I don't anticipate any other modal props but it could still end up getting some in the future, when we implement the actual create functionality and stuff. Might as well leave that option open